### PR TITLE
Update require-capitalized-comments.js

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: lib/rules/require-capitalized-comments.js [hunk 1]
```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```
Comment (TheSavior) [inline lib/rules/require-capitalized-comments.js]: Why is this necessary?

Comment (XhmikosR) [inline lib/rules/require-capitalized-comments.js]: Because without it an error is thrown. The property needs to be quoted.

Comment (TheSavior) [inline lib/rules/require-capitalized-comments.js]: Ah yes, json, not js. Ignore me. ;)

Comment (XhmikosR) [inline lib/rules/require-capitalized-comments.js]: No worries, it took me a while to spot it myself just to realize it was a
bad copy paste from the docs.

Comment (markelog) [inline lib/rules/require-capitalized-comments.js]: Although it is not fully documented but we can accept config declared through js format

**Comment by markelog** [comment]
Thank you

File: lib/rules/require-capitalized-comments.js (hunk 1 above)
Comment (XhmikosR) [inline lib/rules/require-capitalized-comments.js]: Isn't this exact comment used for the rules page? I looked at the website
source and came to the conclusion this file was used.

Comment (markelog) [inline lib/rules/require-capitalized-comments.js]: You are correct, but in order to update the website we need to pull these changes manually, we will release 1.13.1 today and i will update site with it
